components/appbar/overview.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Overview
+page_title: AppBar Overview
+description: Overview of the AppBar component for Blazor.
+slug: appbar-overview
+tags: telerik,blazor,appbar,navbar
+published: True
+position: 0
+---
+
+# Blazor AppBar Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/appbar" target="_blank">Blazor AppBar component</a> helps you build navigation bars for your application seemingly. This article explains the available features.
+
+## Creating Blazor AppBar
+
+1. Add the `<TelerikAppBar>` tag to a Razor file.
+1. Use the `<AppBarSection>` child tag to add content to the AppBar component.
+(optional) Use [content dividers](#content-dividers) to add visual distinction between the sections in the AppBar.
+1. (optional) Set the [position](#positioning) of the AppBar.
```

Comment:
Each step must be reflected in the code example afterwards. Either remove the step, or add the setting in the code.

Reply:
We have no good way to showcase the Postion/PositionMode in code snippets without iframe. So I will remove the step :)